src/components/course/course-header/CourseImportantDates.jsx

```diff
@@ -92,8 +92,9 @@ const CourseImportantDates = () => {
     const soonestExpiringAllocatedAssignment = courseMetadata.availableCourseRuns.find(
       (courseRun) => courseRun.key === soonestExpiringAssignment?.contentKey,
```

Comment:
tiny nit: could remove the `?.` since we're in a conditional where `soonestExpiringAssignment` is known to exist.